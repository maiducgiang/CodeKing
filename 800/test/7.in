10
65
26 5 2 21 15 19 52 52 18 24 21 50 43 64 7 49 43 50 55 48 14 34 55 8 14 57 18 33 26 65 44 62 33 49 48 52 59 25 10 13 23 14 40 38 22 57 44 39 30 17 64 44 61 57 32 43 19 19 46 27 58 57 49 51 62 
60
29 19 54 50 38 58 12 2 41 37 20 46 58 29 53 8 51 7 32 49 48 27 7 22 3 59 55 38 9 27 7 55 13 22 50 40 27 60 22 54 26 48 31 24 59 26 16 59 55 45 33 51 40 21 52 50 41 54 50 10 
52
24 38 24 22 25 49 42 42 39 27 43 6 20 6 33 40 12 34 23 26 16 5 11 40 8 46 2 5 22 50 41 3 3 16 14 6 47 22 1 15 13 41 17 3 41 14 32 36 49 2 21 28 
41
15 41 12 20 19 32 19 12 38 32 19 23 14 14 34 12 41 20 36 15 2 1 27 30 11 7 22 16 19 8 36 26 21 23 13 25 27 30 23 31 9 
46
36 21 5 26 20 31 37 28 7 25 13 32 35 38 26 8 35 6 43 36 31 24 31 4 19 35 9 16 28 39 19 41 46 32 43 24 37 1 42 2 2 1 35 15 46 21 
45
13 13 10 40 12 34 34 27 7 34 41 27 42 2 27 12 6 1 23 16 44 29 14 8 18 22 31 5 15 16 11 42 14 9 20 16 39 31 25 38 15 14 16 6 5 
40
18 17 10 13 38 5 12 39 20 13 26 23 14 17 38 33 13 30 2 28 2 5 30 37 10 38 35 24 24 13 12 37 7 32 6 13 4 35 25 17 
40
32 15 36 12 27 18 20 18 7 12 8 2 10 40 8 18 40 17 20 32 5 8 22 10 25 26 19 2 8 13 25 32 2 20 4 31 37 18 25 26 
48
10 35 29 42 46 21 20 24 47 40 25 47 5 27 44 6 24 1 22 15 45 16 43 5 13 2 17 18 34 42 3 7 46 25 19 5 27 33 35 22 3 36 45 46 15 5 38 3 
40
30 32 2 22 6 29 16 29 7 11 5 37 16 4 8 25 25 18 29 15 26 27 15 25 37 14 19 39 10 38 30 23 30 12 33 26 34 12 13 16 
