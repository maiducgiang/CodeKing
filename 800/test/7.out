30
24
15
22
5
19
14
18
18
18
